app/controllers/admin/topics_controller.rb

```diff
@@ -61,9 +61,6 @@ def destroy
       @topic.destroy
       flash[:success] = 'Topic deleted'
       redirect_to admin_topics_path
-    else 
-      flash[:danger] = 'You can not delete topic with questions!'
-      redirect_to admin_topics_path  
     end
```

Comment:
@DimaSmagulov - just add code back - in fact we won't use this condition but code should let's be consistent
(there is 0.001% possibility that we can will call this action from other view in future)
@OrestF - for "rest client"  this code should be changed